src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.tsx

```diff
@@ -80,7 +83,9 @@ const EditableSectionBlocks = <T extends EditableBlock = EditableBlock>({
           <div
             key={block.id}
             id={`editableSectionBlocks-${block.id}`}
-            className="govuk-!-margin-bottom-9"
+            className={classNames('govuk-!-margin-bottom-9', {
+              openSectionBlock: openedCommentIds.includes(block.id),
```

Comment:
You need to import the styles and use it here. Have you tested that this PR works?